packages/logs/src/boot/startLogs.ts

```diff
@@ -63,30 +67,47 @@ export function doStartLogs(
   }
 
   function reportError(error: RawError) {
-    errorLogger.error(
-      error.message,
-      combine(
-        {
-          date: error.startClocks.timeStamp,
-          error: {
-            kind: error.type,
-            origin: error.source,
-            stack: error.stack,
-          },
+    const messageContext: Partial<LogsEvent> = {
+      date: error.startClocks.timeStamp,
+      error: {
+        kind: error.type,
+        origin: error.source,
+        stack: error.stack,
+      },
+    }
+    if (error.resource) {
+      messageContext.http = {
+        method: error.resource.method as any, // TODO: Set method to upper case (it's a breaking change),
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏not sure to get the intent of adding the todo, it does not seem related to the current PR and we already have a task for it

Reply:
The todo is here to explain why there is a cast. I can replace it by an other comment like:
`// Cast resource method because case mismatch cf issue RUMF-1152`
or let the cast unexplained?

Reply:
oh, I missed the introduction of the cast compared to previous version.
your new comment seems clearer to me 👍